requirements/prod.in

```diff
@@ -38,10 +38,10 @@ jq==1.4.0
 lxml==4.9.2  # Needed as a top-level dep so that it's available for BeautifulSoup, which doesn't explicitly pull it in
 Markdown==3.4.1
 https://github.com/mozmeao/mdx_outline/archive/refs/tags/markdown-3.4-compatibility.tar.gz#egg=mdx_outline
-markupsafe==2.0.1
+# markupsafe==2.1.2
```

Comment:
Intentional?

Reply:
Ah yeah, we don't need to hard-pin it now and I was experimenting. Will remove it properly